gather_istio

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 MUST_GATHER=${MUST_GATHER:-/usr/bin/openshift-must-gather}
+BASE_COLLECTION_PATH="/must-gather"
```

Comment:
upstream defaults to `$PWD/must-gather`, is this intentional?

Reply:
Must-gather actually hardcodes /must-gather in scripts and specifies $PWD/must-gather in others. I reached out to their developers and they said scripts should be outputting to /must-gather as shown in the scripts

Reply:
https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_network_logs#L2
https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_audit_logs#L2
https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_network_logs#L2